quantum/nodes/diagram-output/diagram-output.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const util = require('util');
+const snippets = require('../../snippets');
+const shell = require('../../python').PythonShell;
+
+module.exports = function(RED) {
+
+  function DiagramOutputNode(config) {
+    RED.nodes.createNode(this, config);
+    const node = this;
+
+    node.on('input', async (msg, send, done) => {
+      console.log(`diagram-output: ${msg}`);
```

Comment:
I think we should remove the `console.log` statement here since it isn't really necessary to output the diagram to the log, and might make things harder to debug if we're populating the console with unhelpful information.